synapse/replication/http/send_event.py

```diff
@@ -33,11 +32,12 @@
 
 
 @defer.inlineCallbacks
-def send_event_to_master(client, host, port, requester, event, context,
+def send_event_to_master(clock, client, host, port, requester, event, context,
                          ratelimit, extra_users):
     """Send event to be handled on the master
 
     Args:
+        clock (Clock)
```

Comment:
it would be nice to try and get the types for these right, to help editors implement clickthrough etc. I believe this is a `synapse.util.Clock` ?

Reply:
done